### DISCUSSION TITLE
Repos activated, but "There was an error completing a request with the server" in the Renovate Dashboard

### DISCUSSION THREAD

**Discussion by theurichde** [renovatebot/renovate#23467] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

Hi there! 

When opening the Mend Renovate Dashboard, we get `There was an error completing a request with the server` for freshly activated (15 hours ago) repositories. We activated those repositories in the Renovate GitHub App, and they appear in the Renovate Dashboard, but we get this error and Renovate does not create an onboarding PR for the affected repositories.

Some examples:

* https://developer.mend.io/github/idealo/sam-aiven-iac
* https://developer.mend.io/github/idealo/sam-utils
* https://developer.mend.io/github/idealo/sam-prometheus-ecs
* https://developer.mend.io/github/idealo/sam-actions-common

There are no logs available.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by PhilipAbed** [comment]
We have some issues for newly activated repos.
We are working on a fix asap.

**Comment by PhilipAbed** [comment]
@theurichde 
Hi. 
We deployed a partial fix. Please check if the issue persists.
By partial i mean there was no job created for your repo yet. you will probably need to press `Run Renovate` button to create a job.

**Reply by theurichde** [reply]
Hi @PhilipAbed 

I can confirm that the error is gone, thanks for the fix! But the onboarding job did not run for the affected repositories. Is a renovate internal scheduling running some when soonish again?

**Reply by PhilipAbed** [reply, marked as answer]
I think you will need to run it manually using the new button
